charm-slurmctld/src/charm.py

```diff
@@ -599,9 +599,13 @@ def _infludb_info_action(self, event):
         influxdb_info = self._get_influxdb_info()
 
         if not influxdb_info:
-            influxdb_info = "not related"
+            info = "not related"
+        else:
+            # Juju does not like underscores in dictionaries
```

Comment:
Do you know the reasoning behind this?

Reply:
Operator framework throws a `ValueError` if there are underscores in the dict keys.